jadler-jdk/src/test/java/net/jadler/stubbing/server/jdk/JdkStubHttpServerIntegrationTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2012 - 2015 Jadler contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package net.jadler.stubbing.server.jdk;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import static net.jadler.Jadler.closeJadler;
+import static net.jadler.Jadler.initJadlerUsing;
+import static net.jadler.Jadler.onRequest;
+import static net.jadler.Jadler.port;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Tests that in works without jadler-all.
+ */
+public class JdkStubHttpServerIntegrationTest {
```

Comment:
I do not remember any more. If we have the test in jadler-all, we can remove this one.